.github/workflows/demo.yml

```diff
@@ -13,27 +13,27 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        python: [3.8]
+        python: ["3.10"]
```

Comment:
Adding tests for multiple version is one thing, but here we cannot guarantee anymore that this works for python 3.8. I'd suggest keeping 3.8 :)

Reply:
yeah i agree in this case .. :) 👍🏼